### PR TITLE
Fix theme color setting on startup

### LINKED ISSUE #3278: Napari viewer does not start
## 🐛 Bug

```
/home/czaki/.pyenv/versions/napari/bin/python /home/czaki/Projekty/napari/examples/add_image.py
Traceback (most recent call last):
  File "/home/czaki/Projekty/napari/examples/add_image.py", line 10, in <module>
    viewer = napari.view_image(data.astronaut(), rgb=True)
  File "/home/czaki/Projekty/napari/napari/view_layers.py", line 134, in view_image
    return _make_viewer_then('add_image', args, kwargs)
  File "/home/czaki/Projekty/napari/napari/view_layers.py", line 115, in _make_viewer_then
    viewer = Viewer(**vkwargs)
  File "/home/czaki/Projekty/napari/napari/viewer.py", line 52, in __init__
    self._window = Window(self, show=show)
  File "/home/czaki/Projekty/napari/napari/_qt/qt_main_window.py", line 420, in __init__
    self._qt_window = _QtMainWindow(viewer)
  File "/home/czaki/Projekty/napari/napari/_qt/qt_main_window.py", line 76, in __init__
    self.qt_viewer = QtViewer(viewer, show_welcome_screen=True)
  File "/home/czaki/Projekty/napari/napari/_qt/qt_viewer.py", line 186, in __init__
    self._create_canvas()
  File "/home/czaki/Projekty/napari/napari/_qt/qt_viewer.py", line 286, in _create_canvas
    self.canvas.bgcolor = get_theme(
  File "/home/czaki/.pyenv/versions/napari/lib/python3.8/site-packages/vispy/util/frozen.py", line 17, in __setattr__
    object.__setattr__(self, key, value)
  File "/home/czaki/Projekty/napari/napari/_vispy/vispy_canvas.py", line 78, in bgcolor
    SceneCanvas.bgcolor.fset(self, _value)
  File "/home/czaki/.pyenv/versions/napari/lib/python3.8/site-packages/vispy/scene/canvas.py", line 180, in bgcolor
    self._bgcolor = Color(color).rgba
  File "/home/czaki/.pyenv/versions/napari/lib/python3.8/site-packages/vispy/color/color_array.py", line 389, in __init__
    rgba = _user_to_rgba(color, clip=clip)
  File "/home/czaki/.pyenv/versions/napari/lib/python3.8/site-packages/vispy/color/color_array.py", line 64, in _user_to_rgba
    raise ValueError("Color values must be between 0 and 1 (or use "
ValueError: Color values must be between 0 and 1 (or use clip=True to automatically clip the values).

Process finished with exit code 1
```

## To Reproduce

1. change theme to light
2. checkout 044beba342ef392f4cbed2e8e3a27f27d4799ccb
3. start napari viewer

## Expected behavior

Viewer should open, not fail on data set. 

## Environment

 - Please copy and paste the information at napari info option in help menubar here:

napari: 0.4.11rc3.dev10+g044beba3.d20210830
Platform: Linux-5.11.0-31-generic-x86_64-with-glibc2.2.5
System: Ubuntu 21.04
Python: 3.8.3 (default, Dec  6 2020, 00:47:20)  [GCC 10.2.0]
Qt: 5.15.2
PyQt5: 5.15.4
NumPy: 1.16.5
SciPy: 1.5.1
Dask: 2021.05.0
VisPy: 0.8.0

OpenGL:
  - GL version:  4.6.0 NVIDIA 470.57.02
  - MAX_TEXTURE_SIZE: 32768

Screens:
  - screen 1: resolution 2560x1440, scale 1.0
  - screen 2: resolution 1920x1080, scale 1.0
  - screen 3: resolution 1920x1080, scale 1.0

Plugins:
  - animation: 0.0.2
  - console: 0.0.3
  - scikit-image: 0.4.11rc3.dev10+g044beba3.d20210830
  - svg: 0.1.5  

 - Any other relevant information:

## Additional context

Fast check shows that bug was introduced in #3034

### PR DESCRIPTION
**Pull request by lukasz-migas**
This PR fixes #3278 where the canvas color was incorrectly being set during startup.

### CODE REVIEW

**Comment by Czaki** [comment]
There is an option to add a test?

**Comment by lukasz-migas** [comment]
Sure, will do.

File: napari/_qt/_tests/test_qt_viewer.py [hunk 1]
```diff
@@ -484,3 +485,16 @@ def test_leaks_labels(qtbot, make_napari_viewer):
     assert not gc.collect()
     assert not lr()
     assert not dr()
+
+
+@pytest.mark.parametrize("theme", ("light", "dark"))
```
Comment (Czaki) [inline napari/_qt/_tests/test_qt_viewer.py]:
```suggestion
@pytest.mark.parametrize("theme", available_themes())
```

File: napari/_qt/_tests/test_qt_viewer.py [hunk 2]
```diff
@@ -16,6 +16,7 @@
     layer_test_data,
     skip_local_popups,
 )
+from napari.settings import get_settings
 from napari.utils.interactions import mouse_press_callbacks
 from napari.utils.io import imread
```
Comment (Czaki) [inline napari/_qt/_tests/test_qt_viewer.py:21]:
```suggestion
from napari.utils.io import imread
from napari.utils.theme import available_themes
```

**Review by Czaki — CHANGES_REQUESTED** [review]
Maybe a more generic version?

**Review by sofroniewn — APPROVED** [review]
LGTM!

**Pull request merged**